docs/user_guide/quickstart.rst

```diff
@@ -133,10 +128,26 @@ are placed in the ``build/<design>`` directory.
    tasktime            0.35         1.4           0.83          0.6           0.8           1.71          2.69          0.66          2.2          0.74
 
 
+If you find that having all the tool-specific information printed to the screen is too noisy, you can set the :keypath:`option,quiet` parameter to True.
+
+.. code-block:: bash
+
+   chip.set('option','quiet',True)
+   
+
+Even if you use the :ref:`quiet option <quiet>`, you will still be able to access the tool-specific output from the log file that is saved for each tool. You can find the log files associated with each tool in: ``build/<design>/<jobname>/<tool>/<step>/<logfile>``
```

Comment:
@gadfort make sure you're okay with the file name here. I wanted to be explicit with where the logfiles were located in case the use want to look for them.

Reply:
It should be ``build/<design>/<jobname>/<step>/<index>/<step>.log``